data/map.txt

```diff
@@ -22490,7 +22490,8 @@ system Rutilicus
 	government Republic
 	attributes "dirt belt"
 	arrival 625
-	habitable 625
+	departure 2000
+	habitable 2000
```

Comment:
idk why you change the habitable range here
also I guess this is test data so prob best to remove it from the PR since it isnt a draft

Reply:
whoops my testing is in